IoTHubManager/CreateDeviceTests.cs

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net;
+using System;
+using System.Text;
+using Helpers.Http;
+using Xunit;
+using Newtonsoft.Json.Linq;
+
+namespace IoTHubManager
+{
+    [Collection("IoTHub Manager Tests")]
+    public class CreateDeviceTest
+    {
+
+        internal HttpRequestWrapper Request;
+
+        private string DEVICE_TEMPLATE_AUTO_GEN_AUTH, 
+                       DEVICE_TEMPLATE_SYMMETRIC_AUTH,
+                       DEVICE_TEMPLATE_X509_AUTH;
+
+        public CreateDeviceTest()
+        {
+        
+            //Create request wrapper object for interacting with iothub manager microservices for CRUD on devices
+            this.Request = new HttpRequestWrapper(Constants.Urls.IOTHUB_ADDRESS, Constants.Urls.DEVICE_PATH);
+
+            //Fetch different device templates
+            DEVICE_TEMPLATE_AUTO_GEN_AUTH = System.IO.File.ReadAllText(Constants.Path.DEVICE_FILE_AUTO_GEN_AUTH);
+            DEVICE_TEMPLATE_SYMMETRIC_AUTH = System.IO.File.ReadAllText(Constants.Path.DEVICE_FILE_SYMMETRIC_AUTH);
+            DEVICE_TEMPLATE_X509_AUTH = System.IO.File.ReadAllText(Constants.Path.DEVICE_FILE_X509_AUTH);                     
+        }
+
+        /// <summary>
+        /// Integration test using a real HTTP instance.
+        /// Tests for creation of devices with all permutation of AUTH and Id creation
+        /// </summary>
+        
+
+        /**
+         * Creates a device with auto generated id and symmetric auth.
+         * Verifies that the resultant device has system-generated
+         * id as well as auth credentails.
+         */
+        [Fact, Trait(Constants.TEST , Constants.INTEGRATION_TEST )]
```

Comment:
nit: Formatting seems wrong for all the attributes no space before `,` and not space before `)`. Here all the other places